src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -4730,6 +4776,37 @@ namespace AirflowNetworkBalanceManager {
             state.dataAirflowNetwork->AirflowNetworkCompData(i).CompNum = i;
         }
 
+        // This is also a bit of a hack to keep things working, this needs to be removed ASAP
+        j += state.dataAirflowNetworkBalanceManager->NumOfReliefFans;
+        int ii = 1 + j;
+        int type_i = 1;
+        for (auto &el : state.dataAirflowNetwork->SpecifiedMassFlowData) {
+            state.dataAirflowNetwork->AirflowNetworkCompData(ii).Name = el.name;
+            solver.compnum[el.name] = ii;
+            state.dataAirflowNetwork->AirflowNetworkCompData(ii).CompTypeNum = iComponentTypeNum::SMF;
+            state.dataAirflowNetwork->AirflowNetworkCompData(ii).TypeNum = type_i;
+            state.dataAirflowNetwork->AirflowNetworkCompData(ii).EPlusName = "";
+            state.dataAirflowNetwork->AirflowNetworkCompData(ii).EPlusCompName = "";
+            state.dataAirflowNetwork->AirflowNetworkCompData(ii).EPlusType = "";
+            state.dataAirflowNetwork->AirflowNetworkCompData(ii).CompNum = ii;
+            ++ii;
+            ++type_i;
+        }
+
+        type_i = 1;
+        for (auto &el : state.dataAirflowNetwork->SpecifiedVolumeFlowData) {
+            state.dataAirflowNetwork->AirflowNetworkCompData(ii).Name = el.name;
+            solver.compnum[el.name] = ii;
+            state.dataAirflowNetwork->AirflowNetworkCompData(ii).CompTypeNum = iComponentTypeNum::SVF;
+            state.dataAirflowNetwork->AirflowNetworkCompData(ii).TypeNum = type_i;
+            state.dataAirflowNetwork->AirflowNetworkCompData(ii).EPlusName = "";
+            state.dataAirflowNetwork->AirflowNetworkCompData(ii).EPlusCompName = "";
+            state.dataAirflowNetwork->AirflowNetworkCompData(ii).EPlusType = "";
+            state.dataAirflowNetwork->AirflowNetworkCompData(ii).CompNum = ii;
+            ++ii;
+            ++type_i;
+        }
+
```

Comment:
Another thing that's probably best left alone. This code is necessary for a couple of places that do reverse lookups of objects. "Fortran polymorphism with array indices" would probably be the best way to describe it.